### DISCUSSION TITLE
Update issue from the Windows Store + naming issue

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1741] [Ideas]
At least one Windows user had an error updating Thorium Reader from the Windows Store (no info on the exact error). He therefore installed Thorium Reader from the Github release. 

As a consequence he got two applications installed, one named "Thorium Reader", the other named "Thorium" !

![image](https://user-images.githubusercontent.com/14943186/152845333-53f6a3c5-c8d5-44ca-a2c3-3017d8dcf45f.png)

Therefore he went on using the old version without knowing ...

**Comment by danielweck** [comment]
I am primarily a Mac user, but whenever I go back to Windows to test Thorium etc. I find the "start menu" very confusing, especially Windows 11 (I have duplicates of Visual Studio 2017/2019) The availability of Thorium via Windows Store AND standalone installer follows a quite common deployment pattern, but it certainly can cause confusion :(

**Comment by danielweck** [comment]
Also, the Windows Store UX / UI is super annoying, because it doesn't provide an "uninstall" button. Unless I am mistaken, the user must open the uninstaller panel in the OS settings, find / scroll to Thorium, then choose "uninstall", confirm, etc. Very clunky user experience.
